doc/classes/BakedLightmap.xml

```diff
@@ -81,6 +81,14 @@
 		<member name="use_denoiser" type="bool" setter="set_use_denoiser" getter="is_using_denoiser" default="true">
 			When enabled, a lightmap denoiser will be used to reduce the noise inherent to Monte Carlo based global illumination.
 		</member>
+		<member name="use_grayscale" type="bool" setter="set_use_grayscale" getter="is_using_grayscale" default="false">
+			Only store grayscale values in the lightmap texture. This results in a smaller file size, but all color information will be lost. This can be used if
+			the scene contanis only white lights or you don't mind losing color information in indirect lighting.
```

Comment:
Don't wrap lines in the class reference XML as that creates new paragraphs automatically:

```suggestion
			Only store grayscale values in the lightmap texture. This results in a smaller file size, but all color information will be lost. This can be used if the scene contains only white lights or if you don't mind losing color information in indirect lighting.
```